### PR TITLE
removes obsolete "academicYearTitle" property from academic-year model

### PR DESCRIPTION
**Pull request by stopfstedt**
refs ilios/ilios#3272

### CODE REVIEW

**Review by jrjohnson — CHANGES_REQUESTED** [review]
Should come with a minor API version bump to `3.3` to avoid UI confusion relying on this property without the correct backend version.

**Comment by stopfstedt** [comment]
> Should come with a minor API version bump to `3.3` to avoid UI confusion relying on this property without the correct backend version.

good points. handled. :heavy_check_mark:

**Review by jrjohnson — DISMISSED** [review]

File: addon/models/academic-year.js [hunk 1]
```diff
@@ -1,9 +1,5 @@
 import Model, { attr } from '@ember-data/model';
-import { computed } from '@ember/object';
 
 export default Model.extend({
   title: attr('number'),
```
Comment (jrjohnson) [inline addon/models/academic-year.js:4]: This is a string now.

**Review by jrjohnson — CHANGES_REQUESTED** [review]

File: addon/models/academic-year.js (hunk 1 above)
Comment (stopfstedt) [reply addon/models/academic-year.js]: :heavy_check_mark:

Comment (dartajax) [reply addon/models/academic-year.js]: Is this all sorted out? I'd be glad to merge if so.

Comment (stopfstedt) [reply addon/models/academic-year.js]: @dartajax  this needs the corresponding backend change to land first: https://github.com/ilios/ilios/pull/3306

**Review by jrjohnson — APPROVED** [review]

**Pull request merged**